src/scripts/app.js

```diff
@@ -0,0 +1,67 @@
+const app = angular.module('myIndex', []);
+
+app.controller('indexController', ($scope) => {
+  const index = new invertedIndex();
+
+  $scope.showTable = true;
+  $scope.searchResults = false;
+  $scope.titles = [];
+
+  $scope.uploadFile = (fileName, fileContent) => {
+    $scope.data = {};
+    $scope.docCount = [];
+    if (fileName.toLowerCase().match(/\.json$/)) {
+      const reader = new FileReader();
+      reader.onload = (e) => {
+        let content = JSON.parse(e.target.result);
+        index.createIndex(fileName, content);
+        $scope.data = index.indexMap[fileName];
+
+        for (let fileNo = 0; fileNo < content.length; fileNo++) {
+          $scope.docCount.push(fileNo);
+        }
+
+        $scope.$apply($scope.docCount);
+
+      }
+      reader.readAsText(fileContent);
+
+      $scope.$apply(() => {
+        $scope.titles.push(fileName);
+        $scope.selectedFiles = $scope.selectedFile;
```

Comment:
If `$scope.selectedFiles` is a typo, why is it reassigned to itself? if it's not, I don't see `$scope.selectedFiles` used anywhere else.